src/html.js

```diff
@@ -94,17 +94,20 @@ export default class Html extends PureComponent {
           {_.map(scripts, (script, key) => (
             <script src={script} key={'scripts' + key} charSet="UTF-8" />
           ))}
-
           {/* <!-- Load typekit fonts for twreporter.org domain--> */}
           <script
             dangerouslySetInnerHTML={{
-              __html: `(function(d) {
-                var config = {
-                kitId: 'vlk1qbe',
-                scriptTimeout: 3000,
-                async: true
-              },h=d.documentElement,t=setTimeout(function(){h.className=h.className.replace(/\bwf-loading\b/g,"")+" wf-inactive";},config.scriptTimeout),tk=d.createElement("script"),f=false,s=d.getElementsByTagName("script")[0],a;h.className+=" wf-loading";tk.src='https://use.typekit.net/'+config.kitId+'.js';tk.async=true;tk.onload=tk.onreadystatechange=function(){a=this.readyState;if(f||a&&a!="complete"&&a!="loaded")return;f=true;clearTimeout(t);try{Typekit.load(config)}catch(e){}};s.parentNode.insertBefore(tk,s)
-            })(document);`,
```

Comment:
Question: What makes old method probleming since it seems to load fonts asynchronously too ?

Reply:
Yes, it also loads font asynchronously.

But based on my observation, the bizarre behavior took place after the font is loaded. 

IMO, the issue is about what's the adobe font script does after the font is loaded, which is still not clear why it leads to such bizarre behavior.

For me, it is hard to trace the code `https://use.typekit.net/vlk1qbe.js` since it is minimized and uglified.

I proposed a similar solution in #2148 to address this issue, which uses web font loader and google font instead to mitigate the issue.